migration/src/bin/migrate_to.rs

```diff
@@ -0,0 +1,393 @@
+use clap::{builder::PossibleValuesParser, command, Parser, ValueEnum};
+use migration::{
+    sea_orm::{ConnectOptions, Database, DatabaseConnection, EntityTrait, QueryOrder},
+    MigratorTrait,
+};
```

Comment:
Could these be cleaned up so that we pull them in via `sea_orm_migration` instead of via our local `migration` crate? As is, it's confusing, because it makes it look as though items like `MigrationTrait` are defined in our `migration` crate.